### ISSUE TITLE
Include fees

### ISSUE THREAD

**Issue by GarboMuffin** [GarboMuffin/nicehash-calculator#8]
NiceHash order fee, pools fees, etc. Should be possible to account for some of these at least.

**Comment by tidelake** [comment]
Yes, the order fees are killer, and scaled up!  Can you tell me how I can start working on this? It might take me several days to understand what your code is doing. Maybe you have something small I can start with. :-)

**Comment by GarboMuffin** [comment]
The 3% fee is possible to include. Perhaps just take a 3% cut off of the revenue in NiceHashCalculator.ts before passing it onto a handler. `revenueData.revenue *= 0.97` or something like that.

The 0.0001 BTC fee isn't possible to include here -- maybe more can be added to the header about that.

**Comment by GarboMuffin** [comment]
I added `--experimental-fees` to the rewrite a little while ago. It's experimental and I have no idea if I'm doing it right. Eventually I would want to change it to `--include-fees` or something and also add a `--order-size` or something so that the 0.0001 BTC fee can be accounted for. I do still want to add `--pool-fee`, defaulting to 1%.
